composer.json

```diff
@@ -16,17 +16,17 @@
 		}
 	],
 	"require": {
-		"php": "^5.3.2|~7.0",
-		"symfony/framework-bundle": "~2.3|~3.0"
+		"php": "^5.3.9|~7.0",
```

Comment:
I guess 2.0 can be released soon, yes. But I'd like to keep Symfony 2.7 compatibility.